puppet/modules/quickstack/manifests/neutron/plugins/plumgrid.pp

```diff
@@ -0,0 +1,15 @@
+# Common class used by Plumgrid Neutron Plugin.
+class quickstack::neutron::plugins::plumgrid (
```

Comment:
Actually this class will be called by compute nodes. I thought to put it as a separate common class but now I think it would be better to put in quickstack/manifests/neutron/compute.pp and remove this class, that would be better.